samples/_bootstrap.py

```diff
@@ -0,0 +1,42 @@
+import os
+import sys
+import re
+
+STORAGE_ACCOUNT_NAME = os.environ.get("AZSTORAGETORCH_STORAGE_ACCOUNT_NAME")
+CONTAINER_NAME = os.environ.get("AZSTORAGETORCH_CONTAINER_NAME")
+
+
+def modify_url(path):
+    if not STORAGE_ACCOUNT_NAME or not CONTAINER_NAME:
+        print(
+            "Please set environment variables AZSTORAGETORCH_STORAGE_ACCOUNT_NAME and AZSTORAGETORCH_CONTAINER_NAME"
+        )
+        return
+
+    with open(path, "r") as f:
+        content = f.read()
+
+    pattern = r'(CONTAINER_URL\s*=\s*\(\s*"https://)([^.]+)(\.blob\.core\.windows\.net/)([^"]+)("\s*\))'
+    new_url = rf"\1{STORAGE_ACCOUNT_NAME}\3{CONTAINER_NAME}\5"
```

Comment:
Instead of searching for the `CONTAINER_URL` value, can we instead search for `<my-storage-account-name>` and `<my-container-name>`? This will probably simplify the search pattern and more representative of what we are asking a customer to do (e.g. replace those placeholders)